client/src/components/Button/ButtonRow.js

```diff
@@ -20,7 +20,7 @@ const ButtonRow = ({ title, description, onPress }) => (
 ButtonRow.propTypes = {
   title: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
-  onPress: PropTypes.func.isRequired,
+  onPress: PropTypes.func,
```

Comment:
Because nested items on the button row have onPress, so the Parent(?) having onPress is not always needed. in this instance the ModalSelector handles the onPress.